### DISCUSSION TITLE
Switch settings - Add a Clear or Reset button

### DISCUSSION THREAD

**Discussion by bivvo** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#115] [Editor App Feature Requests]
In the editor after clicking a switch, It would be nice to have clear or reset button settings. I was going in to edit a switch and it would be nice to be able to clear it out before modifying.

**Comment by simonaglover95** [comment]
@bivvo Thanks! We're planning to add a clear/reset switch button soon!
